### PR TITLE
Blog Post: Seventh Wave of Bitcoin Grants

### PR DESCRIPTION
**Pull request by Arvin21M**
**Build Preview:**

- [/blog/bitcoin-grants-september-2024-7th-wave](https://os-website-git-7th-wave-bitcoin-grants-open-sats.vercel.app/blog/bitcoin-grants-september-2024-7th-wave)

**Social Preview:**

- image

### CODE REVIEW

**Comment by Arvin21M** [comment]
@OpenSats/volunteers Team, would some one please mind doing a quick sanity check on this when you have a moment? Would like to publish this Monday or Tuesday this week. 🙏

File: data/blog/seventh-wave-of-bitcoin-grants.mdx [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+[Minibits](https://www.minibits.cash/) is a mobile Cashu wallet that integrates
+Bitcoin and the Lightning Network, offering a seamless payment experience
+through ecash, Lightning addresses, zaps, and Nostr identities. Key features
+include seed-based recovery, Tor network support, LNURL pay and withdraw, and
+encrypted push notifications. The next development phase focuses on iOS support,
+enhanced security, and improved interoperability with the latest ecash and
+Lightning technologies. As a fully open-source wallet, Minibits plays an
+important role in advancing the [Cashu](https://cashu.space/) protocol by
+providing a fast cross-platform mobile wallet which will put Cashu's scalability
+and privacy claims to the test in real-world scenarios
```
Comment (Bayernatoor) [inline data/blog/seventh-wave-of-bitcoin-grants.mdx:87]:
```suggestion
and privacy claims to the test in real-world scenarios.
```

File: data/blog/seventh-wave-of-bitcoin-grants.mdx [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+License: MIT
+
+### DLC Dev Kit
+
+[DLC Dev Kit](https://www.dlcdevkit.com/) is a comprehensive development kit
+that simplifies the integration of Discreet Log Contracts (DLCs) into Bitcoin
+applications, allowing developers to create financial contracts like futures and
+peer-to-peer bets without having to deal with the complexities of DLC logic
+directly. By providing libraries for wallet operations, DLC communication, data
+storage, and Oracle integration, dlcdevkit enables seamless application
```
Comment (Bayernatoor) [inline data/blog/seventh-wave-of-bitcoin-grants.mdx:100]:
```suggestion
storage, and Oracle integration, dlc dev kit enables seamless application
```

File: data/blog/seventh-wave-of-bitcoin-grants.mdx [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+foster a more cohesive and interoperable DLC ecosystem, driving broader adoption
+and enabling developers to build innovative Bitcoin applications with minimal
+friction.
+
+Repository: [bennyhodl/dlcdevkit](https://github.com/bennyhodl/dlcdevkit)  
+License: MIT
+
+### Cove
+
+Cove is a mobile wallet built with BDK. Designed for simplicity and
```
Comment (Bayernatoor) [inline data/blog/seventh-wave-of-bitcoin-grants.mdx:113]:
```suggestion
Cove is a mobile wallet built with [BDK](https://github.com/bitcoindevkit/bdk/blob/master/README.md). Designed for simplicity and
```

**Review by Bayernatoor — CHANGES_REQUESTED** [review]
Left a couple suggestions/corrections.

Besides those, everything looks good and all links are correct/work. 

Good job guys! :clap:

**Comment by btcschellingpt** [comment]
Apologies for the lateness of this feedback .. The reference to AssumeUTXO "in BCA"

Where did "Bitcoin Core App" come from? Is this a reference to the new Bitcoin Core QML Gui (which is implied from the author commit link)? By referencing this as "BCA" it's unclear if this is a new "project" from the developer or part of core (which I think it is).

**Comment by dergigi** [comment]
> Where did "Bitcoin Core App" come from? Is this a reference to the new Bitcoin Core QML Gui (which is implied from the author commit link)?

@btcschellingpt I've added a link which should clear this up:

> This project aims to integrate Assume UTXO functionality into the [Bitcoin Core App](https://bitcoincore.app/) (BCA) [...]

---

The announcement looks good to me, so I'll push this live soon :shipit: 

Thanks for the feedback and the edits everyone 🙏🧡

**Review by dergigi — APPROVED** [review]
Looks good to me ✅

**Review by dergigi — APPROVED** [review]
Looks good to me ✅

**Comment by dergigi** [comment]
Live now!
- https://opensats.org/blog/bitcoin-grants-september-2024-7th-wave
- [nostr post](https://njump.me/nevent1qqsdueqvqfhprzh74jpldr2vl4g6emfpwmnsdnn7407cxqjsfwe3nnspz3mhxue69uhhyetvv9ujuerpd46hxtnfduq3vamnwvaz7tmjv4kxz7fwwpexjmtpdshxuet5qgs8suecw4luyht9ekff89x4uacneapk8r5dyk0gmn6uwwurf6u9rus3mhcnh)

**Pull request merged**